nltk/embedding/word2vec.py

```diff
@@ -0,0 +1,134 @@
+# Natural Language Toolkit: Word2Vec Wrapper through Gensim
+#
+# Copyright (C) 2001-2015 NLTK Project
+# Author: Long Duong <longdt219@gmail.com>
+# URL: <http://nltk.org/>
+# For license information, see LICENSE.TXT
+
+"""
+Word2Vec Wrapping through Gensim. Allowing 
+- Training new model
+- Load the pre-trained model from nltk.corpus 
+- Run some demo tasks  
+"""
+from __future__ import absolute_import, division, unicode_literals
+import gensim
+
+
+
+class Word2Vec():
```

Comment:
Hi @stevenbird, 
I have the similar concern too. Currently, we haven't add any value to the wrapper. We might be able to do something more for the wrapper, e.g. visualization of word embedding using <a href = 'maaten.github.io/tsne/'> t-SNE </a>.